lib/functions/main/rootfs-image.sh

```diff
@@ -56,7 +56,7 @@ function build_rootfs_and_image() {
 	LOG_SECTION="apt_lists_copy_from_host_to_image_and_update" do_with_logging apt_lists_copy_from_host_to_image_and_update
 
 	# creating xapian index that synaptic runs faster
-	if [[ "${BUILD_DESKTOP}" == yes && -f "${SDCARD}/usr/sbin/update-apt-xapian-index" ]]; then
+	if [[ "${BUILD_DESKTOP}" == yes && ${BETA} != "yes" && -f "${SDCARD}/usr/sbin/update-apt-xapian-index" ]]; then
```

Comment:
"${BETA}" to have consistent style

Reply:
thanks @igorpecovnik corrected with commit 3d3ff0b0012309f9119f659ce6df66823b846c48